### PR TITLE
expose Runner and get access to transform results

### PR DESCRIPTION
**Pull request by ganarajpr**
Access to the runner from the jscodeshift api - so that you can programmatically run jscodeshift ( same as running it from cli ). Currently I am not sure how you can do this without this PR. 

This seems to have been requested before ( #47 ) and perhaps was implemented but seems to have been missed recently ?

### CODE REVIEW

**Comment by joyarzun** [comment]
Hi there
Can you please review this PR? For now it's the best option to call Jscodeshift without cli.
Thanks

**Comment by ganarajpr** [comment]
@fkling Any update on this?

File: src/Collection.js [hunk 1]
```diff
@@ -118,6 +118,15 @@ class Collection {
     return this.__paths.length;
   }
 
+  /**
+   * Provides a callback mechanism to be notified when done with previous steps.
+   *
+   * @param {function} callback
+   */
+  done(cb) {
```
Comment (fkling) [inline src/Collection.js]: Which use case does this solve? Why not do

```
Collection.fromNodes(...).forEach(...);
// do stuff afterwards
```

The `Collection` API is not asynchronous.

**Comment by buoyantair** [comment]
@ganarajpr Are you still interested in working on this? It would be really nice to have some way to run transformations & get status etc.

**Pull request closed without merging**